mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -26,24 +25,61 @@ fun PlayScreen(
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(),
 ) {
+  val buttonState = rememberExpandableFloatingActionButtonState()
+
   Box(modifier = modifier.fillMaxSize().padding(contentPadding)) {
-    NewGameButton(onNewGame = state.onNewGameClick, modifier = Modifier.align(Alignment.BottomEnd))
+    NewGameButton(
+        state = buttonState,
+        onLocalGame = { state.onLocalGameClick() },
+        onRemoteGame = { state.onOnlineGameClick() },
+        modifier = Modifier,
+    )
   }
 }
 
 /**
  * Composable that composes a new game button
- * @param onNewGame callable upon button action
+ * @param onLocalGame callable upon button action
  * @param modifier the [Modifier] for this composable.
  */
 @Composable
-fun NewGameButton(onNewGame: () -> Unit, modifier: Modifier = Modifier) {
+fun NewGameButton(
+    state: ExpandableFloatingActionButtonState,
+    onLocalGame: () -> Unit,
+    onRemoteGame: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
   val strings = LocalLocalizedStrings.current
-  ExtendedFloatingActionButton(
-      text = { Text(strings.newGame) },
-      onClick = onNewGame,
-      shape = CircleShape,
-      modifier = modifier.padding(16.dp).height(56.dp),
-      backgroundColor = PawniesColors.Green800,
-      icon = { Icon(PawniesIcons.Add, null) })
+
+  Box(
+      modifier.fillMaxSize().background(MaterialTheme.colors.background),
+      Alignment.BottomEnd,
+  ) {
+    ExpandableFloatingActionButton(
+        state = state,
+        expandedContent = {
+          ExpandableFloatingActionButtonItem(
+              onClick = onLocalGame,
+              icon = { Icon(PawniesIcons.LocalPlay, null) },
+              text = { Text(strings.prepareGamePlayLocal) },
+              modifier = Modifier.fillMaxWidth(),
+          )
+
+          Divider(color = MaterialTheme.colors.onPrimary)
+
+          ExpandableFloatingActionButtonItem(
+              onClick = onRemoteGame,
+              icon = { Icon(PawniesIcons.OnlinePlay, null) },
+              text = { Text(strings.prepareGamePlayOnline) },
+              modifier = Modifier.fillMaxWidth(),
+          )
+        },
+        collapsedContent = {
+          Icon(PawniesIcons.Add, null)
+          Spacer(Modifier.width(16.dp))
+          Text("New game".uppercase())
```

Comment:
Should come from localized strings.

Reply:
Fixed in [fdcc578](https://github.com/epfl-SDP/android/pull/226/commits/fdcc5788b387441425278a14bcb33bfb8aa7cb3d)